bridge_ui/src/contexts/EthereumProviderContext.tsx

```diff
@@ -222,7 +221,6 @@ export const EthereumProviderProvider = ({
           projectId: WALLET_CONNECT_PROJECT_ID,
           showQrModal: false,
           chains: [getEvmChainId(CHAIN_ID_ETH) as number, getEvmChainId(CHAIN_ID_BSC) as number],
-          rpcMap: EVM_RPC_MAP,
```

Comment:
We no longer need to set the `rpcMap` because WalletConnect has a default RPC URL for each chain. Also, the RPC URLs in the bridge UI no longer work.